customer-account-extension/src/OrderStatusBlock.liquid

```diff
@@ -0,0 +1,67 @@
+{%- if flavor contains "react" -%}
+import {
+  BlockStack,
+  reactExtension,
+  TextBlock,
+  Link,
+  Banner,
+  useApi
+} from "@shopify/ui-extensions-react/customer-account";
+
+export default reactExtension(
+  "customer-account.order-status.block.render",
+  () => <PromotionBanner />
+);
+
+function PromotionBanner() {
+  const { i18n } = useApi();
+
+  return (
+    <Banner>
+      <BlockStack inlineAlignment="center" >
+        <TextBlock>
+          {i18n.translate("earnPoints")}
+          <Link>
+            {i18n.translate("viewRewards")}
+          </Link>
+        </TextBlock>
+      </BlockStack>
+    </Banner>
+  );
+}
+
+
+{%- else -%}
+import { extension, Banner, BlockStack, TextBlock, Link } from '@shopify/ui-extensions/customer-account';
+
+export default extension('customer-account.order-status.block.render', (root, api) => {
+  const { i18n } = api;
+
+  const banner = root.createComponent(
+    Banner,
+    {}
+  )
+
+  const block = root.createComponent(
+    BlockStack,
+    {inlineAlignment: "center"}
+  )
+
+  const text = root.createComponent(
+    TextBlock,
+    {},
+    i18n.translate("earnPoints")
+  )
+
+  const link = root.createComponent(
```

Comment:
If we decide to remove the link, we need to do it here too.

Reply:
Link is removed.